tensorflow/lite/examples/label_image/get_top_n_impl.h

```diff
@@ -30,19 +30,29 @@ extern bool input_floating;
 template <class T>
 void get_top_n(T* prediction, int prediction_size, size_t num_results,
                float threshold, std::vector<std::pair<float, int>>* top_results,
-               bool input_floating) {
+               int input_type) {
   // Will contain top N results in ascending order.
   std::priority_queue<std::pair<float, int>, std::vector<std::pair<float, int>>,
                       std::greater<std::pair<float, int>>>
       top_result_pq;
 
   const long count = prediction_size;  // NOLINT(runtime/int)
+  float value = 0.0;
+
   for (int i = 0; i < count; ++i) {
-    float value;
-    if (input_floating)
-      value = prediction[i];
-    else
-      value = prediction[i] / 255.0;
+    switch (input_type) {
+      case kTfLiteFloat32:
+        value = prediction[i];
+        break;
+      case kTfLiteInt8:
+        value = (prediction[i] + 128) / 256.0;
```

Comment:
Why is this 256 while the uint8 case is 255?

Reply:
This is to dequantize values in the final op. For uint8 cases, as far as I can tell, most models generated with quantized-aware training have value between 0 and 255.  

The last layer of the int8 [Mobilenet V3 EdgeTPU model](https://storage.cloud.google.com/mobilenet_edgetpu/checkpoints/mobilenet_edgetpu_224_1.0.tgz) I used to test this, has scale = 0.00390625 = 1 / 256.